### DISCUSSION TITLE
İbax referral

### DISCUSSION THREAD

**Discussion by Mami493065** [IBAX-io/go-ibax#60] [Q&A] [answered]
Friends, I invited 14 people and I invited all of them correctly, but it seems that I was invited to 9 people. Whoever I invite, someone I invited before is deleted from the list. None of the people I invite are fake. All of them are real accounts, I invited someone more today, another friend I invited before was deleted. please solve this problem of mine

**Comment by yolandadadada** [comment, marked as answer]
If your friend unfollows IBAX during the airdrop, then it will disappear from your referral list.

**Reply by DVanGuruB** [reply]
github is ok. but twitter isnt

**Reply by hsynd58** [reply]
anladım

**Comment by Mami493065** [comment]
My friends did not stop following ibax, they all continue to follow ibax,
but they were deleted from the part I invited

19 Oca 2022 Çar 10:01 tarihinde yolandadada ***@***.***> şunu
yazdı:

**Reply by yolandadadada** [reply]
can you give me your Twitter username? Let me check for you.

**Comment by Mrt58** [comment]
@Murat58guler Twitter

19 Oca 2022 Çar 11:18 tarihinde yolandadada ***@***.***> şunu
yazdı:

**Comment by yolandadadada** [comment]
The system shows that you have not been successfully referred.

**Comment by Mrt58** [comment]
github is ok. but twitter is not rt

19 Oca 2022 Çar 11:48 tarihinde DVanGuruB ***@***.***> şunu
yazdı:

**Reply by DVanGuruB** [reply]
twitter is ok.

**Comment by Mrt58** [comment]


**Reply by DVanGuruB** [reply]
ı stopped trying

**Comment by Mami493065** [comment]
@Muhamme50728325

19 Oca 2022 Çar 11:18 tarihinde yolandadada ***@***.***> şunu
yazdı:

**Comment by Mrt58** [comment]
My Twitter name is @Murat58guler  I want to update my wallet address.

19 Oca 2022 Çar 11:18 tarihinde yolandadada ***@***.***> şunu
yazdı:

**Comment by hrhrrcp** [comment]
hrhrrcp

iPhone’umdan gönderildi

MURAT GÜLER ***@***.******@***.***>> şunları yazdı (19 Oca 2022 17:27):


My Twitter name is @Murat58guler I want to update my wallet address.

19 Oca 2022 Çar 11:18 tarihinde yolandadada ***@***.***> şunu
yazdı:

> can you give me your Twitter username? Let me check for you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/IBAX-io/go-ibax/discussions/60#discussioncomment-1995882>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUV2JUMEHVNZBZYG6ISGEALUWZXUHANCNFSM5MGRUZVQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/60#discussioncomment-1998242>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXL3UY52IQMTZFGKQNDPT53UW3C6ZANCNFSM5MGRUZVQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by hemresenol** [comment]
If it is a fake account, it will not accept it as a reference, please check it properly.
